### PR TITLE
RTCM3 SBP Converter

### PR DESCRIPTION
**Pull request by pgrgich**
This is a new branch based on test_framework (see PR https://github.com/swift-nav/piksi_buildroot/pull/156) that merges in changes from the master. @anth-cole @jacobmcnamee

### CODE REVIEW

File: host_tests/sbp_rtcm3_bridge_tests/sbp_rtcm_converter_tests.c [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+  msg1006_expected.msg_1005.arp_y = -5576347.7863;
+  msg1006_expected.msg_1005.arp_z = 2578377.2472;
+  msg1006_expected.ant_height = 0.0;
+
+  sbp_to_rtcm3_1006( &sbp_base_pos, &msg1006_out );
+
+  assert( msg1006_equals( &msg1006_expected, &msg1006_out ) );
+}
+
+// pgrgich: code snippet that casts incoming SBP message to a struct so it can be viewed in the debugger.
```
Comment (anth-cole) [inline host_tests/sbp_rtcm3_bridge_tests/sbp_rtcm_converter_tests.c]: Do we need this commented code?

File: package/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 2.8.7)
+project(piksipackages)
+
+SET(CMAKE_MODULE_PATH "cmake" )
+SET(LIBZMQ_LIBRARIES "libzmq" )
+SET(LIBZMQ_INCLUDE_DIRS "/home/pgrgich/piksi_buildroot/package/third_party/libzmq/include" )
```
Comment (anth-cole) [inline package/CMakeLists.txt]: we should make these relative paths.

File: package/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+add_subdirectory(third_party/libsbp/c)
+add_subdirectory(third_party/libzmq)
+add_subdirectory(third_party/czmq)
+
+
+
+add_subdirectory(zmq_adapter/src)
+add_subdirectory(sbp_rtcm3_bridge/src)
+
+# include(cmake/SanitizeTargets.cmake)
```
Comment (anth-cole) [inline package/CMakeLists.txt]: can we get rid of commented code?

File: host_tests/sbp_rtcm3_bridge_tests/sbp_rtcm_converter_tests.c (hunk 1 above)
Comment (pgrgich) [inline host_tests/sbp_rtcm3_bridge_tests/sbp_rtcm_converter_tests.c]: Fixed ind55a093.

File: package/CMakeLists.txt (hunk 2 above)
Comment (pgrgich) [inline package/CMakeLists.txt]: Fixed ind55a093.

File: package/CMakeLists.txt (hunk 3 above)
Comment (pgrgich) [inline package/CMakeLists.txt]: Fixed in d55a093.

**Comment by anonymous-travis-swiftnav** [comment]
## v1.0.0-branch-94-g4fab16b
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-94-g4fab16b/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.0.0-branch-94-g4fab16b/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-94-g4fab16b

**Comment by anonymous-travis-swiftnav** [comment]
## v1.0.0-branch-97-g895136f
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-97-g895136f/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.0.0-branch-97-g895136f/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-97-g895136f

**Comment by pgrgich** [comment]
@anth-cole @mfine @jacobmcnamee @gsmcmullin can you review this before I do the merge?

**Comment by anth-cole** [comment]
We still need @Jacob to add the dynamic week number support

**Comment by anonymous-travis-swiftnav** [comment]
## v1.0.0-branch-98-gd400ffd
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-98-gd400ffd/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.0.0-branch-98-gd400ffd/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-98-gd400ffd

**Comment by anonymous-travis-swiftnav** [comment]
## v1.0.0-branch-108-gc1a3313
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-108-gc1a3313/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.0.0-branch-108-gc1a3313/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-108-gc1a3313

**Comment by anonymous-travis-swiftnav** [comment]
## v1.0.0-branch-109-ge1724e1
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-109-ge1724e1/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.0.0-branch-109-ge1724e1/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-109-ge1724e1

**Pull request merged**